app/controllers/locations/ips_controller.rb

```diff
@@ -6,12 +6,13 @@ def new
   end
 
   def create
-    @ip = @location.ips.new(ip_params)
-    if @ip.save
+    if ips.all?(&:valid?)
```

Comment:
This doesn't feel like the "right" flow in a Rails context, but it's also not unclear (I hope).